spec.html

```diff
@@ -485,7 +501,7 @@ <h1>Tree Grammar</h1>
   // or the first in `ExportSpecifier :: IdentifierName as IdentifierName`.
   attribute IdentifierExpression name;
   // The second `IdentifierName` in `ExportSpecifier :: IdentifierName as IdentifierName`, if present.
-  attribute IdentifierName? exportedName;
+  attribute IdentifierReference? exportedName;
```

Comment:
given this is the name of exported thin from this module, this shouldn't be a reference, but either declaration or just property name.